app/views/partials/client/item.blade.php

```diff
@@ -1,7 +1,7 @@
 <td>{{ $client->authority['name'] }}</td>
-<td>{{ chunk_split($client->api['basic_key'], 20, '</br>') }}</td>
-<td>{{ chunk_split($client->api['basic_secret'], 20, '</br>') }}</td>
-<td>
+<td>{{ chunk_split($client->api['basic_key'], 40, '') }}</td>
```

Comment:
Glad you picked these up.
Now that the changes are targeted for 1.4.3, I'll be happy to make all the adjustments required to do these changes properly. I'll update my changes to use CSS where required, and investigate the removal of chunck-split and test before submitting again.